tests/unit/test_validate_attribute.py

```diff
@@ -1574,3 +1558,644 @@ def test__get_column_names(
     ) -> None:
         """Tests for ValidateAttribute._get_column_names"""
         assert va_obj._get_column_names(DataFrame(input_dict)) == expected_dict
+
+    ##############
+    # get_no_entry
+    ##############
```

Comment:
Instead of grouping by comment like this should this be a new class?

```
class TestGetNoEntry:
```

Reply:
Yeah I'm not sure. Do other egineers have preferences for this?